subsys/partition_manager/CMakeLists.txt

```diff
@@ -55,6 +55,16 @@ if (CONFIG_NRF53_UPGRADE_NETWORK_CORE)
   ncs_add_partition_manager_config(pm.yml.pcd)
 endif()
 
+if (CONFIG_BUILD_WITH_TFM)
+  ncs_add_partition_manager_config(pm.yml.tfm)
+
+  # Add the mcuboot hex file built by TFM as the contents of the bl2 partitions,
+  # so it can be flashed together with the app/tfm hex files.
+  set_property(GLOBAL PROPERTY
+    bl2_PM_HEX_FILE ${CMAKE_BINARY_DIR}/tfm/bin/bl2.hex
```

Comment:
Using `${CMAKE_BINARY_DIR}/tfm/bin` is a bit deep knowledge here.
Could we not get this info by other means ?

Reply:
I'm experimenting with adding paths as target properties on `tfm`, for the native ns pr. What do you think of that solution?

Reply:
https://github.com/zephyrproject-rtos/trusted-firmware-m/pull/34

Reply:
I'll leave it as is, since the change won't get downstream in time.

Reply:
accepted.